docs/cue-lists/theatre-programming.md

```diff
@@ -112,72 +66,24 @@ recorded)*. To record everything with intensity above zero, switch
 
 ---
 
-````xml
-<Record> {blue fader button} {blue fader button}
-````
-
-Record the next cue onto the end of the cue list on this fader
-
----
-
-````xml
-<Record> <Cue> n <Enter>
-````
-
-Record cue **n** *(will prompt for merge/overwrite if it exists already)*
-
----
-
-````xml
-<Time> t <Enter>
-````
-
-Set fade time into programmer *(does not
-affect stored cues; will save with the cue when you record it)*
-
----
-
-````xml
-<Cue> n <Time> t <Enter>
-````
-
-Set fade time of cue **n** to **t** seconds
-
----
-
-````xml
-<Delete> <Cue> n <Enter> <Enter>
-````
-
-Delete cue **n** *(2^nd^ \<Enter\> confirms)*
-
----
-
-````xml
-<Copy> <Cue> n <@> <Cue> m <Enter>
-````
-
-Copy cue **n** to new cue **m**
-
----
-
-````xml
-<Copy> <Cue> n <@> <Enter>
-````
-
-Copy cue **n** onto the end of the cue list
+Keypresses | Result
+---- | ----
+\<Record\> {blue fader button} {blue fader button} | Record the next cue onto the end of the cue list on this fader
```

Comment:
We refer to this in a few different ways in the manual:

- "blue fader button"
- "blue button"
- Swop
- swop

We should try be consistent (I appreciate this is the theatre programming page)

Reply:
Add the notorious "select", "Select" and the occasional "handle button". 100% d'accord, but I don't have a solution.

Maybe a very short section on this - maybe in conjunction with key profiles. And then, if possible, a special notation which is parsed into a formatted thingy like \< ... \> for hard keys and \[ ... \] for softkeys/menubuttons. Maybe we could use \{ curly braces \} and have this rendered with blue background? 

As for the name, maybe 'Select/Swop' would be the best solution

Reply:
Ok we'll shelve this conversation and maybe make it an issue that can be resolved across the manual

Reply:
I've open #189 to continue this discussion